src/routes/NavBar.svelte

```diff
@@ -12,60 +13,122 @@
+                    href="/people/">People</a
+                >
+            </li>
+            <li>
+                <a
+                    class="transition-colors duration-300 ease-in-out hover:text-csi-blue"
+                    href="/events/">Events</a
+                >
+            </li>
+            <li>
+                <a
+                    class="transition-colors duration-300 ease-in-out hover:text-csi-blue"
+                    href="/projects/">Projects</a
+                >
+            </li>
+        </ul>
+    </nav>
+    <div class="flex items-center gap-2">
+        <button
+            class="rounded-full border border-csi-neutral-200 p-2 transition-colors duration-200 hover:bg-light-gray hover:shadow-inner dark:border-csi-neutral-800 dark:hover:bg-dark-gray"
+            on:click={theme.toggle}
+        >
+            <Icon src={Moon} size="18px" theme="solid" class="dark:hidden" />
+            <Icon src={Sun} size="18px" theme="solid" class="hidden dark:block" />
+        </button>
+        <a
+            href="mailto:lino@up-csi.org"
+            class="hidden whitespace-nowrap rounded-full border-2 border-csi-blue px-4 py-1 transition-colors duration-300 hover:bg-csi-blue hover:shadow-inner lg:block"
+        >
+            Contact Us
+        </a>
+
+        <button
+            class="rounded-full border border-csi-neutral-200 p-2 transition-colors duration-200 hover:bg-light-gray lg:hidden dark:border-csi-neutral-800 dark:hover:bg-dark-gray"
+            on:click={() => (mobileMenu = !mobileMenu)}
+        >
+            {#if mobileMenu}
+                <Icon src={XMark} theme="solid" class="hover:text-csi-blue" size="18px" />
+            {:else}
+                <Icon src={Bars3} theme="solid" size="18px" />
+            {/if}
+        </button>
+    </div>
+</div>
 {#if mobileMenu}
-    <div class="flex h-svh flex-col bg-blue-white dark:bg-black">
-        <div class="fixed bottom-0 w-full space-y-4 pb-12">
-            <ul class="w-full font-dm text-2xl text-csi-black dark:text-csi-white">
-                <li class="m-4"><a href="/" on:click={closeMenu}>Home</a></li>
-                <li class="m-4"><a href="/about/" on:click={closeMenu}>About</a></li>
-                <li class="m-4"><a href="/people/" on:click={closeMenu}>People</a></li>
-                <li class="m-4"><a href="/events/" on:click={closeMenu}>Events</a></li>
-                <li class="m-4"><a href="/projects/" on:click={closeMenu}>Projects</a></li>
-                <li class="m-4"><button on:click={closeMenu}>Get in Touch</button></li>
-            </ul>
-            <div class="flex w-full items-center justify-center">
+    <div
+        class="fixed inset-y-2 right-2 block h-[calc(100vh-1rem)] w-5/6 rounded-2xl bg-blue-white/95 font-dm text-xl backdrop-blur-2xl sm:w-3/4 sm:max-w-sm lg:hidden dark:bg-black/95 dark:text-csi-white"
+        transition:fly={{ x: 300, duration: 150 }}
+    >
```

Comment:
```suggestion
    <div
        class="fixed inset-2 block rounded-2xl bg-blue-white/95 font-dm text-xl backdrop-blur-2xl sm:left-auto sm:w-3/4 sm:max-w-sm lg:hidden dark:bg-black/95 dark:text-csi-white"
        transition:fly={{ x: 300, duration: 150 }}
    >
```
I don't think `h-[calc(100vh-1rem)]` would do anything with both `fixed` and `inset-y-2` set here.

That aside, similar to what I've mentioned in a prior review, I think the mobile NavBar still looks awkward on small screens, even with `w-5/6`. These changes make it so that it instead extends to the other side of the screen for smaller screens while keeping the slight margin of `0.5rem`.

Reply:
`sm:left-auto` was actually the property I was looking for. But I don't use `-auto` that much which is why I settled for w-5/6. Will surely use this instead. Thanks!